src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -78,15 +78,24 @@ public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface
         $routeName = $this->getRouteName($resourceClass, false);
 
         $identifierValues = [];
+        $identifierValuesComposite = [];
+
+        $propertyNames = [];
         foreach ($this->propertyNameCollectionFactory->create($resourceClass) as $propertyName) {
             $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $propertyName);
 
             if ($propertyMetadata->isIdentifier()) {
                 $identifierValues[] = $this->propertyAccessor->getValue($item, $propertyName);
+                $identifierValuesComposite[] = $propertyName.'='.$this->propertyAccessor->getValue($item, $propertyName);
```

Comment:
Just change the above line to:

```php
$identifiers[$propertyName] = $this->propertyAccessor->getValue($item, $propertyName);
```

Then do this after the loop:

```php
if (count($identifiers) > 1) {
    $identifiers = array_map(function ($identifierName, $identifierValue) {
        return sprintf('%s=%s', $identifierName, $identifierValue);
    }, array_keys($identifiers), $identifiers);
}
```

And finally:

```php
return $this->router->generate($routeName, ['id' => implode(',', $identifiers)], $referenceType);
```